classifier/cpu.go

```diff
@@ -46,9 +47,35 @@ func checkCPU(profile *hwcc.HardwareClassification, host *bmh.BareMetalHost) boo
 		return false
 	}
 
+	ok = checkArch(
```

Comment:
I suggest to change the method name checkArch to checkCPUArch

Reply:
Done